docs/whats_new.rst

```diff
@@ -12,6 +12,220 @@ This page keeps a detailed human friendly rendering of what's new and changed
+        - Remove :func:`Client.edit_command_permissions`, :func:`Client.bulk_edit_command_permissions`, :func:`Client.edit_command_permissions`, :func:`Client.edit_command_permissions`
+        - Remove :func:`Guild.get_command_permissions`, :func:`Guild.edit_command_permissions`, :func:`Guild.bulk_edit_command_permissions`
+        - Update behavior of :class:`GuildCommandInteraction` annotation to automatically set ``dm_permission=False`` instead of adding a local check, remove support for subcommands
+        - Add :class:`ApplicationCommandPermissionType` enum, change type of :attr:`ApplicationCommandPermissions.type` to support channel targets
+        - |commands| Remove :func:`~ext.commands.guild_permissions` decorator
+        - |commands| Remove ``sync_permissions`` parameter from :class:`~ext.commands.Bot`
+    - New features:
+        - Add ``dm_permission`` and ``default_member_permissions`` parameters to application command objects and decorators
+        - Add :attr:`~ApplicationCommand.dm_permission`, :attr:`~ApplicationCommand.default_member_permissions` attributes
+          to :class:`ApplicationCommand` and :class:`~ext.commands.InvokableApplicationCommand`
+        - Add :func:`ApplicationCommandPermissions.is_everyone` and :func:`ApplicationCommandPermissions.is_all_channels`
+        - Add :attr:`AuditLogAction.application_command_permission_update` enum value and :attr:`AuditLogDiff.command_permissions`
+        - Add :func:`on_application_command_permissions_update` event
+        - |commands| Add :func:`~ext.commands.default_member_permissions` decorator, alternative to identically named parameter
+- Add guild previews (:issue:`359`)
+    - Add :class:`GuildPreview`
+    - Add :func:`Client.fetch_guild_preview`
+- Add guild widget settings and widget url (:issue:`360`, :issue:`365`)
+    - Add :class:`WidgetSettings`, :class:`WidgetStyle`
+    - Add :func:`Guild.widget_settings`, :func:`Guild.widget_image_url`
+    - Add :func:`Widget.image_url`
+    - Change :func:`Guild.edit_widget` return type
+- Add guild welcome screens (:issue:`339`)
+    - Add :class:`WelcomeScreen`, :class:`WelcomeScreenChannel`
+    - Add :func:`Guild.welcome_screen`, :func:`Guild.edit_welcome_screen`
+    - Add :attr:`Invite.guild_welcome_screen`
+
+- Support ``List[str]`` and ``Dict[str, str]`` in ``option`` parameter of :class:`disnake.ui.Select` (:issue:`326`)
+- Add :func:`Guild.search_members` (:issue:`358`, :issue:`388`)
+- Add :attr:`ModalInteraction.message` (:issue:`363`, :issue:`400`)
+- Support :func:`InteractionResponse.edit_message` for modal interactions, if modal was sent in response to component interaction (:issue:`364`, :issue:`400`)
+- Support ``reason`` parameter in :func:`Message.create_thread` and :func:`Thread.delete` (:issue:`366`)
+- Add :attr:`StageInstance.guild_scheduled_event` and :attr:`StageInstance.guild_scheduled_event_id` (:issue:`394`)
+- Add :attr:`Client.session_start_limit` (:issue:`402`)
```

Comment:
and SessionStartLimit

Reply:
574b5ed1ab4209e1f11e206fc22a872793f276ba